flyteplugins/go/tasks/pluginmachinery/flytek8s/k8s_resource_adds.go

```diff
@@ -17,6 +17,9 @@ import (
 
 const (
 	flyteExecutionURL = "FLYTE_EXECUTION_URL"
+
+	FlyteInternalWorkerNameEnvVarKey        = "_F_WN"  // "FLYTE_INTERNAL_WORKER_NAME"
+	FlyteInternalDistErrorStrategyEnvVarKey = "_F_DES" // "FLYTE_INTERNAL_DIST_ERROR_STRATEGY"
```

Comment:
```suggestion
	flyteInternalDistErrorStrategyEnvVarKey = "_F_DES" // "FLYTE_INTERNAL_DIST_ERROR_STRATEGY"
```

Reply:
This can't be done, right? These are used by the plugin, which is in a different package.